doc/devdocs/modules/launcher/plugins/windowssettings.md

```diff
@@ -0,0 +1,108 @@
+# Windows Settings Plugin
+
+The Windows settings Plugin allows users to search the Windows settings.
+
+## Special functions (differ from the regular functions)
+
+* Support modern Windows settings (Windows 10+)
+* Support legacy Windows settings (Windows 7, 8.1)
+* Support extra programs for setting (like ODBC)
+
+* Support search by the are of the setting (like `Privacy`)
+* Support search by the command line of the setting (like `workplace` found `ms-settings:workplace` )
```

Comment:
Do we support this? That's new for me. 🤔

Reply:
This isn't implemented. Except the command is added to the `altNames` list.

Reply:
oh your are right, I wonder why I have remove this from the original PR.
we can add this later in a separate PR, when we need it.